src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -299,6 +303,19 @@ export default {
 		}
 
 		this.initializeFiveDollarsNotes();
+
+		const userBalance = Number(this.userInfo?.balance) ?? 0;
```

Comment:
This needs to be `Number(this.userInfo?.balance ?? 0)` instead, since `Number(undefined)` is `NaN`, and `NaN ?? 0` is `NaN`.

Reply:
nice call!
balance is coming in a string, for validation we need a number I think. It's still working b/c NaN is false when validating. any tho?

Reply:
`??` is null coalescing. So `NaN ?? 0` equates to `NaN.

The line should be: `const userBalance = Number(this.userInfo?.balance ?? 0);`

Reply:
thank you!